test/inductor/test_triton_kernels.py

```diff
@@ -27,6 +27,11 @@
     import triton
     from triton import language as tl
 
+    from triton.language.extra.cuda.libdevice import (
```

Comment:
do you need to import this at top level? could you do it inside the particular test?

Reply:
No, Triton compiler can't pick it up when the import is inside the test, unfortunately.